cmd/eksctl-anywhere/cmd/upgradecluster.go

```diff
@@ -98,6 +99,14 @@ func (uc *upgradeClusterOptions) upgradeCluster(cmd *cobra.Command) error {
 		return fmt.Errorf("failed to build cluster manager opts: %v", err)
 	}
 
+	var skippedValidations map[string]bool
+	if len(uc.skipValidations) != 0 {
```

Comment:
I'd love to get all these validations out of the command function, but it's not something we should worry about for this.